fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1755,4 +1755,13 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean recursive_dir_search_enabled = false;
+
+    @ConfField
+    public static String ssl_keystore_location = "";
+
+    @ConfField
+    public static String ssl_keystore_password = "";
+
+    @ConfField
+    public static String ssl_key_password = "";
```

Comment:
Will this information be exposed in web UI or other metrics interface?

Reply:
This is an enterprise-edition  feature. So these configs will not exposed in our community docs.

Reply:
I mean `http://<FE_HOST>:8030/variable`